dataservice/api/workflow/models.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.ext.associationproxy import association_proxy
+
+from dataservice.extensions import db
+from dataservice.api.common.model import Base
+from dataservice.api.genomic_file.models import GenomicFile
+
+
+class Workflow(db.Model, Base):
+    """
+    Workflow entity represents an executed bioinformatics pipeline
+
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param created_at: Time of object creation
+    :param modified_at: Last time of object modification
+    :param task_id: Id of executed task
+    :param name: Name of workflow
+    :param version: Version of workflow
+    :param github_url: URL to repository hosted on GitHub
+    """
+    __tablename__ = 'workflow'
+
+    task_id = db.Column(db.Text())
+    name = db.Column(db.Text())
+    version = db.Column(db.Text())
+    github_url = db.Column(db.Text())
```

Comment:
This library looks pretty cool: http://sqlalchemy-utils.readthedocs.io/en/latest/_modules/sqlalchemy_utils/types/url.html

We could probably use it for custom types like URLs and maybe even do some validation at the model level. Something to look into later